### PR TITLE
Pull request with comments in English

### PR DESCRIPTION
**Pull request by eduardoazevedo3**

### CODE REVIEW

**Comment by plribeiro3000** [comment]
@eduardoazevedo3 LGTM. My only question is about the `guia_sindical` folder.
Is that config going to be used by all versions/banks?

**Comment by eduardoazevedo3** [comment]
@plribeiro3000 `guia_sindical` is exclusive to Caixa Econômica Federal, the other banks are not authorized to issue this.

**Comment by plribeiro3000** [comment]
So this change make it possible to parse `Caixa Econômica Federal` cnabs?
Maybe we should name it before that instead?

**Comment by eduardoazevedo3** [comment]
I do not know what name I could give to this folder. `guia_sindical` is an exclusive resource for trade unions, it does not follow the same `boleto` rule.

**Comment by plribeiro3000** [comment]
@eduardoazevedo3 Got it. After reading a lot about it [here](http://www.caixa.gov.br/Downloads/contribuicao-sindical-urbana-leiaute/Contribuicao-Sindical-Urbana-Leiaute-CNAB-240-Remessa-Retorno.pdf) and other places i believe i got it.

What you called `guia sindical` is the layout implementation of CNAB 240 for `Caixa Econômica` so we should rename it to `caixa_economica` instead and update the README.

I've just noticed you changed the `santander` layout for `segmento T` as well. This is a non backwards compatible change. Could you remove it please?

Another thing to note is that you've added implementations of `segmento Y` for the others but they are incomplete. I rather leave them without support and check this in the initialization code [here](https://github.com/plribeiro3000/cnab/pull/2/files#diff-253c00c3819d0f53224ee4dfb3b8924dR33).

**Comment by eduardoazevedo3** [comment]
@plribeiro3000 I suggest to refuse this pull request, I'm finishing homologating the `guia_sindical` along with the manager of the `Caixa Econômica Federal`, everything is fine, I'll create another pull request for you.

**Comment by plribeiro3000** [comment]
Its ok. If it works, just update your branch and do a pull force to rewrite the commits in this PR. =)

No need to open a new one. 😬

**Comment by plribeiro3000** [comment]
BTW, i'm really excited to have support for `Caixa Econômica` here. =)

So i will try to push this forward and help you as much as i can.

I really appreciate your work on this. Thank you very much!

**Comment by eduardoazevedo3** [comment]
All right

File: lib/cnab/line.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Line
     include PrettyInspect
 
     def initialize(line, definition)
-      raise Exceptions::LineNotParseable unless line.size == 242
+      raise Exceptions::LineNotParseable if line.size != 241 and line.size != 242
```
Comment (plribeiro3000) [inline lib/cnab/line.rb:6]: This condition will never trigger, right?

Comment (eduardoazevedo3) [reply lib/cnab/line.rb:6]: I don't know why I did it, it sure is wrong.

**Pull request closed without merging**